helix-core/src/test/java/org/apache/helix/integration/common/ZkStandAloneCMTestBase.java

```diff
@@ -94,6 +95,8 @@ public void beforeClass() throws Exception {
 
   @AfterClass
   public void afterClass() throws Exception {
+    String testClassName = this.getShortClassName();
```

Comment:
Can we add it to ZkTestBase? The @AfterClass methods with different names will be executed one by one.

Reply:
Let me remove it as @afterclass of zktest base do print out something like"

```
AfterClass: TestAutoRebalance called
```